plugins/woocommerce-blocks/assets/js/editor-components/search-list-control/style.scss

```diff
@@ -177,7 +177,6 @@
 			cursor: pointer;
 
 			&::after {
-				background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6z" fill="#{encode-color($gray-700)}" /></svg>');
```

Comment:
I moved this to something more specific (see below) because it wasn’t specific enough to be applied when needed.